cmake/std/atdm/utils/set_build_options.sh

```diff
@@ -133,6 +135,10 @@ elif [[ $ATDM_CONFIG_BUILD_NAME == *"clang-5.0.1"* ]]; then
   export ATDM_CONFIG_COMPILER=CLANG-5.0.1
 elif [[ $ATDM_CONFIG_BUILD_NAME == *"clang"* ]]; then
   export ATDM_CONFIG_COMPILER=CLANG
+elif [[ $ATDM_CONFIG_BUILD_NAME == *"xl-2019.08.20"* ]]; then
+  export ATDM_CONFIG_COMPILER=XL-2019.08.20
+elif [[ $ATDM_CONFIG_BUILD_NAME == *"xl"* ]]; then
+  export ATDM_CONFIG_COMPILER=XL
```

Comment:
FYI: If you have a custom_builds.sh file, then this never gets exectued.  Therefore, I would take this out.